### ISSUE TITLE
how to speed up the first page load, then load the whole app later on asynchronously ?

### ISSUE THREAD

**Issue by jpca999** [meanjs/mean#1634]
Any ideas would be welcomed.

**Comment by hyperreality** [comment]
As far as I know, Angular doesn't support this. It's not the point of a SPA. I did hear Angular 2 may have made steps towards this, but I'm no Angular 2 expert so not sure where to point you.

**Comment by jpca999** [comment]
I understand that,  here's an idea load the first page via nginx or some server and only when other pages are called for load the rest of the app, this is how big Corp e commence sites do stuff but I was thinking if there's a simple way to do that.  Coz almost 90% of the customers don't even go past the first page load...

**Comment by shanavas786** [comment]
A simple solution would be to use a non-angular (or angular with only required dependencies) index page and provide a link to application page from it which loads the complete app.

**Comment by hyperreality** [comment]
@shanavas786 Good idea. You could even use an identical static bootstrap navbar to keep the same look and feel.
